iOS/Flows/Conversation/Cells/ConversationMessagesCell.swift

```diff
@@ -218,6 +214,28 @@ class ConversationMessagesCell: UICollectionViewCell, ConversationUIStateSettabl
                                     itemsToReconfigure: itemsToReconfigure,
                                     showLoadMore: self.shouldShowLoadMore)
             }.store(in: &self.subscriptions)
+
+        ConversationsManager.shared.$activeConversation
+            .removeDuplicates()
+            .mainSink { activeConversation in
+
+                // If this cell's conversation becomes active,
+                // then start message consumption if needed.
+                guard activeConversation?.cid == self.conversation?.cid else { return }
+
+                for cell in self.collectionView.visibleCells {
+                    guard let messageCell = cell as? MessageCell,
+                          let indexPath = self.collectionView.indexPath(for: messageCell),
+                          messageCell.areDetailsShown else { continue }
+
+                    guard let item = self.dataSource.itemIdentifier(for: indexPath),
+                          case .message(let cid, let messageID, _) = item else { continue }
+
+                    let message = ChatClient.shared.message(cid: cid, id: messageID)
+
+                    self.startConsumptionIfNeeded(for: message, at: indexPath)
+                }
+            }.store(in: &self.subscriptions)
```

Comment:
Listen for changes to the active conversation to determine if we need to start consuming messages.

This fixes issues where sliding to a new conversation didn't start consuming a message until you scrolled.